packages/core/src/definitions/components.ts

```diff
@@ -1,12 +1,21 @@
 import type { ComponentDefinition } from '@/types';
-
-import { CONTENTFUL_COMPONENT_CATEGORY, CONTENTFUL_CONTAINER_ID } from '@/constants';
-import { containerBuiltInStyles } from './styles';
+import * as Constants from '@/constants';
```

Comment:
probably not a big deal here, but import * can cause havok on tree shaking. I kinda like being able to see whats being used in a file as well by glancing at the imports. If we are running into issues where the imports are exploding (i know they are in some places, we can look into some refactoring of the constants to help).